garden/src/pages/HomePage.tsx

```diff
@@ -1,11 +1,231 @@
-import React from 'react'
+import React from "react";
+import Navbar from "../components/Navbar";
+import { Link } from "react-router-dom";
+import GardenBox from "../components/GardenBox";
 
 const HomePage = () => {
   return (
     <>
-      <div>HomePage</div>
+      {/* <div className="container-max-w-full bg-gray-200 shadow border p-6">
+        HomePage
+      </div> */}
+      <Navbar />
+      <div className="font-display flex flex-col items-center pt-5 sm:pt-10 px-5">
+        <p className="font-semibold text-3xl sm:text-6xl text-center max-w-3xl">
+          Build a garden where your model can thrive
+        </p>
+        <p className="pt-8 sm:pt-16 text-lg sm:text-3xl text-center max-w-lg">
+          An ecosystem of ML models and datasets ready to be explored
+        </p>
+        {/* <button className="mt-12 bg-green w-48 h-[64px] rounded-xl text-white">
+          Search
+        </button> */}
+        <Link
+          to="/search"
+          className="mt-12 bg-green w-48 h-[64px] rounded-xl text-white text-center flex justify-center items-center hover:saturate-150"
+        >
+          Search
+        </Link>
+      </div>
+
+      <div
+        id="icons-text"
+        className="grid grid-cols-3 place-items-top gap-4 pt-14 px-[10%] font-display text-sm sm:text-2xl "
+      >
+        {/* chart */}
+        <div className="flex flex-col items-center">
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
```

Comment:
I'm not familiar with `xmlns`- do we need to update this to our website? or is this like a citation of where you got the svg?

Reply:
That came by default directly from Heroicons, and it stands for xml namespace. It's not required in html documents so there is no problem in taking it out if you think that would be best.